machine_learning_hep/processerdhadrons_jet.py

```diff
@@ -401,65 +451,69 @@ def process_efficiency_single(self, index):
-            h_gen_fd = TH1F("h_gen_fd" + stringbin2, "FD Generated in acceptance |y|<0.5", \
+            h_gen_fd = TH1F("h_gen_fd" + stringbin2, "FD Generated in acceptance |y|<0.5",
                             self.p_nptfinbins, self.var1binarray)
-            h_presel_fd = TH1F("h_presel_fd" + stringbin2, "FD Reco in acc |#eta|<0.8 and sel", \
+            h_presel_fd = TH1F("h_presel_fd" + stringbin2, "FD Reco in acc |#eta|<0.8 and sel",
                                self.p_nptfinbins, self.var1binarray)
-            h_sel_fd = TH1F("h_sel_fd" + stringbin2, "FD Reco and sel in acc |#eta|<0.8 and sel", \
+            h_sel_fd = TH1F("h_sel_fd" + stringbin2, "FD Reco and sel in acc |#eta|<0.8 and sel",
                             self.p_nptfinbins, self.var1binarray)
 
             bincounter = 0
             for ipt in range(self.p_nptfinbins):
                 bin_id = self.bin_matching[ipt]
                 if self.do_custom_analysis_cuts:
-                    df_mc_reco = pickle.load(openfile(self.mptfiles_recosk[bin_id][index], "rb"))
-                    df_mc_reco["imp_par_prod"] = df_mc_reco["imp_par_prod"].astype(float) # allow negative cut values
+                    df_mc_reco = pickle.load(
+                        openfile(self.mptfiles_recosk[bin_id][index], "rb"))
+                    df_mc_reco["imp_par_prod"] = df_mc_reco["imp_par_prod"].astype(
+                        float)  # allow negative cut values
                 else:
-                    df_mc_reco = pickle.load(openfile(self.mptfiles_recoskmldec[bin_id][index], "rb"))
+                    df_mc_reco = pickle.load(
+                        openfile(self.mptfiles_recoskmldec[bin_id][index], "rb"))
                 df_mc_reco = adjust_nsd(df_mc_reco)
                 if self.s_evtsel is not None:
                     df_mc_reco = df_mc_reco.query(self.s_evtsel)
                 if self.s_jetsel_reco is not None:
-                    if self.p_usejetptbinned_deff is True:
-                        df_mc_reco = df_mc_reco.query(self.s_jetsel_reco)
+                    df_mc_reco = df_mc_reco.query(self.s_jetsel_reco)
                 if self.s_trigger is not None:
                     df_mc_reco = df_mc_reco.query(self.s_trigger)
                 if self.runlistrigger is not None:
-                    df_mc_reco = selectdfrunlist(df_mc_reco, \
-                             self.run_param[self.runlistrigger], "run_number")
-                df_mc_gen = pickle.load(openfile(self.mptfiles_gensk[bin_id][index], "rb"))
+                    df_mc_reco = selectdfrunlist(df_mc_reco,
+                                                 self.run_param[self.runlistrigger], "run_number")
+                df_mc_gen = pickle.load(
+                    openfile(self.mptfiles_gensk[bin_id][index], "rb"))
                 df_mc_gen = adjust_nsd(df_mc_gen)
-                if self.p_usejetptbinned_deff is True:
+                if self.s_jetsel_gen is not None:
```

Comment:
Why is the `p_usejetptbinned_deff` condition removed?

Reply:
same as above